source/installation/deps/windows.html.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Dependencies on Windows
+---
+
+There are several packages that need to be installed before getting started compiling
+Plume on Windows devices. This guide will only cover installations for SQLite, with 
+adding PostgreSQL in the future.
+
+1. Download and install [Git-SCM](https://git-scm.com/downloads)
+2. Download and install [Microsoft Visual Studio 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=Community&rel=16)
+3. Install [Chocalatey](https://chocolatey.org/) using a PowerShell v2+ prompt:
+
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+```
+
+4. Download and install [OpenSSL](https://slproweb.com/products/Win32OpenSSL.html) (I installed to C:\Extra\ssl)
```

Comment:
Rephrasing to avoid "I" would be better IMO.

Reply:
I did think about this when I did my initial writing up. I will add a comment above the line instead of inline.